### ISSUE TITLE
Imputed value not missing (Nan)

### ISSUE THREAD

**Issue by mrtrunghieu1** [spindro/GINN#3]
I run code in examples so I think GINN imputed value not missing (Nan value). Can you explain problems to me?

**Comment by spindro** [comment]
Hi,
GINN is part of a family of algorithms whose objective is the imputation of missing values.
Basically, you have a dataset which is incomplete (i.e. some of the observations have missing elements) and use GINN, or other algorithms, to fill these holes. After this step, you can continue to perform any task that you want with the data.

**Comment by Hu-nie** [comment]
Are you saying that you are learning through complete data and imputation missing data afterwards?

**Comment by spindro** [comment]
Not exactly. The task here is to learn from the incomplete data the values that most likely should fill the gaps in the dataset. After this imputation step, you have a new dataset with the predicted values instead of NaNs or whatever you use as a placeholder.

**Comment by mrtrunghieu1** [comment]
How do you evaluate your imputation algorithm? So In my opinion, I have
full dataset then missing value by missingness 10%,20%... then evaluate
classification in them

Vào 20:27, Th 4, 13 thg 1, 2021 Indro Spinelli <notifications@github.com>
đã viết:

**Comment by Hu-nie** [comment]
In my opinion, you should generate random missing values with complete data and compare and evaluate them with MSE, RMSE, MAE, etc.

**Comment by hemarathore** [comment]
@Hu-nie But how to evaluate MSE, ...and other matrices? we don't have the missing values, or the ground truth. You are saying this model is filling your missing data and that's it, but how to access the performance measures?
